### PR TITLE
Add xfce status report.

### PR DESCRIPTION
**Pull request by madpilot78**

### CODE REVIEW

**Review by pauamma — CHANGES_REQUESTED** [review]
Some (like the link syntax) are mechanical and I could make them while or after merging,  but others are best left to your appreciation.

File: 2022q4/xfce.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+Contact: Guido Falsi <madpilot@FreeBSD.org>
+
+The FreeBSD Xfce team (xfce@) works to ensure the Xfce desktop environment
+is maintained and fully functional on FreeBSD.
+
+This quarter the Xfce team members are pleased to welcome Xfce 4.18
+to the FreeBSD ports tree!
+
+This new release includes many improvements in various parts of the
+environment, with many improvements in the Thunar file manager.
```
[the remark this answers is not in the document]
Comment (madpilot78) [reply 2022q4/xfce.md:16]: Thanks, should have reread it before submitting!

File: 2022q4/xfce.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Also various upstream releases now include patches that were present in
```
[the remark this answers is not in the document]
Comment (madpilot78) [reply 2022q4/xfce.md:18]: To tell the truth I was using "release" a a substantive.

But there is no problem changing that.

File: 2022q4/xfce.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+## Xfce on FreeBSD ##
+
+Links: +
+link:https://xfce.org/about/news/?post=1671062400[Xfce 4.18 Upstream Release Announcement] URL: link:https://xfce.org/about/news/?post=1671062400[https://xfce.org/about/news/?post=1671062400] +
+link:https://www.freshports.org/x11-wm/xfce4[Xfce meta-port on FreshPorts] URL: link:https://www.freshports.org/x11-wm/xfce4[link:https://www.freshports.org/x11-wm/xfce4]
```
Comment (pauamma) [inline 2022q4/xfce.md:5]:
```suggestion
link:https://www.freshports.org/x11-wm/xfce4[Xfce meta-port on FreshPorts] URL: link:https://www.freshports.org/x11-wm/xfce4[https://www.freshports.org/x11-wm/xfce4]
```

**Review by pauamma — APPROVED** [review]
Minor nit, fixable while/after merging.

File: 2022q4/xfce.md (hunk 3 above)
Comment (madpilot78) [reply 2022q4/xfce.md:5]: Thanks!

**Review by pauamma — APPROVED** [review]

**Pull request merged**